packages/block-library/src/classic/edit.js

```diff
@@ -64,6 +62,12 @@ export default class ClassicEdit extends Component {
 	initialize() {
 		const { clientId } = this.props;
 		const { settings } = window.wpEditorL10n.tinymce;
+
+		// Prevent multiple inits on the same classic block
+		if ( this.editor && this.editor.initialized ) {
```

Comment:
Why are we calling `initialize` again in the first place on an already initialised block?

Reply:
@tofumatt initialize being called on page load was the main issue, so we need to initialize them in a more "on demand" fashion to get around it. Perhaps an alternate here is to disable the `onFocus` handler after the block has been initialized so we wouldn't need this check?